src/store.ts

```diff
@@ -119,7 +119,8 @@ const updateProject = (
 };
 
 export interface State {
-  gestures: GestureData[];
+  // TODO: Rename store gestures as actions instead
+  gestures: ActionData[];
```

Comment:
This affects the key used in localStorage for storing action data. Renaming to be done in a separate PR as suggested by Matt.